.werft/installer-tests.ts

```diff
@@ -18,9 +18,10 @@ const version: string = annotations.version || "-";
 const preview: string = annotations.preview || "false"; // setting to true will not destroy the setup
 const upgrade: string = annotations.upgrade || "false"; // setting to true will not KOTS upgrade to the latest version. Set the channel to beta or stable in this case.
 const skipTests: string = annotations.skipTests || "false"; // setting to true skips the integration tests
+const selfSigned: string = annotations.selfSigned || "false";
```

Comment:
`SelfSigned` annotation passed via job makes sense, but when running cron, this doesn't get toggled automatically, so I recommend doing a randomize on [true, false] for this boolean. So it rotates for tests.

```suggestion
const selfSigned: string = annotations.selfSigned || randomize(["true", "false"]);
```